### DISCUSSION TITLE
Help identifying the cause of an assert

### DISCUSSION THREAD

**Discussion by hardyb** [visgl/deck.gl#6612] [General]
I am on this version:
    "deck.gl": "^7.2.6",

I am intermittently getting an assert failure in layer.js
  _initState() {
    assert(!this.internalState && !this.state);
...

I think this occurs during the call to...
new GeoJsonLayer(...);

I realize this is not a lot to go on, but what are the most common causes of this assert?

Many thanks.

**Comment by Pessimistress** [comment]
If you are debugging in development mode, then you can see a comment next to the assert telling you what it's about. You cannot reuse the same layer instance after it's been removed.
